packages/account/src/Sections/Security/TwoFactorAuthentication/digit-form.tsx

```diff
@@ -1,57 +1,70 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Formik, Form, Field } from 'formik';
+import { Formik, Form, Field, FormikHelpers, FieldProps, FormikProps } from 'formik';
 import { Input, Button } from '@deriv/components';
 import { getPropertyValue, WS } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 
-const DigitForm = ({ is_enabled, setTwoFAStatus, setTwoFAChangedStatus, is_language_changing }) => {
+export type TDigitForm = {
```

Comment:
please dont export prop types.
you can use `React.ComponentProps<typeof DigitForm >` in test files or wherever you are importing this.

Reply:
Okay